cli/data_preparation/waymo/create_data.py

```diff
@@ -102,6 +120,12 @@ def _fill_infos(root_path, frames, split="train", nsweeps=1):
                         "token": frame_name,
                         "transform_matrix": None,
                         "time_lag": 0,
+                        "veh_to_global": ref_obj["veh_to_global"],
+                        "gt_boxes": None,
+                        "gt_names": None,
+                        "difficulty": None,
+                        "name": None,
+                        "num_points_in_gt": None
                     }
```

Comment:
gt_boxes, gt_names, diffuculty, num_points_in_gt should be put in an "annotations" dict same as key-frame. 
L178 has saved gt_boxes as annotations, why put them here again?